tox.ini

```diff
@@ -1,5 +1,12 @@
 [tox]
-envlist = py27,py36
+envlist = py27,py35,py36,py37
```

Comment:
again, would 3.5 and 3.6 be enough at the moment?

Reply:
see above